src/components/filter/filter-date/FilterDate.tsx

```diff
@@ -37,6 +41,33 @@ export const FilterDate = ({
 
   const [editingFilterText, setEditingFilterText] = useState<string>('');
 
+  // adjusts the date to set the filter to be inclusive of the entered value
+  const getAdjustedDate = ({
+    date,
+    type,
+  }: {
+    date: string | null;
+    type: 'begin' | 'end';
+  }) => {
+    switch (rangeType) {
+      case 'is between':
+      case 'is equal to':
+        if (type === 'begin') {
+          return dayjs(date).subtract(1, 'days').format(defaultDateFormat);
+        }
+        if (type === 'end') {
+          return dayjs(date).add(1, 'days').format(defaultDateFormat);
+        }
+        return date;
+      case 'is on or after':
+        return dayjs(date).subtract(1, 'days').format(defaultDateFormat);
+      case 'is before or on':
+        return dayjs(date).add(1, 'days').format(defaultDateFormat);
+      default:
+        return date;
+    }
+  };
```

Comment:
<img width="442" alt="image" src="https://github.com/user-attachments/assets/f3cd1f94-5d5d-496a-9b58-814682ea10e0" />
Looks like `dateEnd` is not working right 🤔